src/service.js

```diff
@@ -443,6 +443,20 @@ module.exports = function(mixinOptions) {
 				}
 			},
 
+			/**
+			 * Return the field name in a GraphQL Mutation, Query, or Subscription declaration
+			 * @param {String} declaration - Mutation, Query, or Subscription declaration
+			 * @returns {String} Field name of declaration
+			 */
+			getFieldName(declaration) {
+				// Remove all multi-line/single-line definitions and comments
```

Comment:
Just a slight change for clarity sake -- the things being removed are `descriptions and comments` rather than `definitions and comments`